enh(release): prepare engine 20.10.5 release notes (#779)
centreon/centreon-documentation: en/releases/centreon-core.md

## Centreon Engine

### 20.10.5

> 

Reviewer: ```suggestion

> This version requires Centreon Clib and Centreon-Connector versions to be 20.10.2 or higher.

```

#### Bugfixes

> - Engine burns 100% of the CPU when check_period set to none

Reviewer 2: ```suggestion
- Engine cpu usage increased to 100% when `check_period` is set to `none`
```

## Centreon Engine

### 20.10.5

> > This version requires Centreon Clib and Centreon-Connector versions to be 20.10.2 or higher.

Reviewer: ```suggestion
> This version requires Centreon Clib version to be 20.10.2 or higher.
> This version requires Centreon-Connector version to be 20.10.1 or higher.
```

## Centreon Engine

### 20.10.5

> 

Reviewer 2: ```suggestion

`June 4, 2021`

```
